unittests/AST/TensorFlow.cpp

```diff
@@ -77,9 +82,9 @@ class TypeContainsTensorFlowValueTest : public ::testing::Test {
 TEST_F(TypeContainsTensorFlowValueTest, ClassifiesCorrectly) {
```

Comment:
Given that you have expanded the test, why don't you change the name of the test fixture? Say TensorTypeUtilsTest?

Reply:
yes, that makes sense. Will rename it.